src/query/src/sql.rs

```diff
@@ -182,6 +183,35 @@ pub async fn show_tables(
                 .context(error::CreateRecordBatchSnafu)?;
             Ok(Output::RecordBatches(records))
         }
+        ShowKind::Full => {
+            let mut table_types = Vec::with_capacity(tables.len());
+            for table_name in &tables {
+                let table_type = catalog_manager
+                    .table(query_ctx.current_catalog(), &schema_name, table_name)
+                    .await
+                    .context(error::CatalogSnafu)?
+                    .unwrap()
+                    .table_type();
```

Comment:
suggestion: We should skip this table it is `None` instead of using `unwrap()`. It is possible that the table has been dropped before we get the table type.